src/components/elements/input/MinutesDurationInput.tsx

```diff
@@ -80,8 +77,11 @@ const MinutesDurationInput = ({
           helperText={null}
           fullWidth={false}
           placeholder='0'
-          ariaLabelledBy={getLabelText() + ' (hours)'}
```

Comment:
@martha whoops sorry I missed this on the previous review. I hit an error (linked) so I put in a quick fix for it. 

Here i'm setting `aria-label` on the actual input component. The aria label is the text that would be read by the screen reader. In the example in the storybook story it would be `Duration (hours)` and `Duration (minutes)` for each input.

`aria-labelledby` would have been doable too, but this seemed simpler. For aria-labelledby, you pass it the _id_ of the element that labels it -- you don't pass it the label itself, which is what you were doing above. sorry I missed that! https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label both pretty good explanations

Reply:
Thanks this is much better, yeah I was suspicious of that `getLabelText` function I wrote and glad we can expunge it!